src/App.vue

```diff
@@ -147,6 +147,11 @@ export default class App extends Mixins(DateMixin, FilingTemplateMixin, LegalApi
       .find(x => x.entityType === val) : null)
   }
 
+  // Bound to the successful login event in SigninView
+  private redirectOnLogin (): void {
+    alert('in app')
```

Comment:
Ok for debugging but don't commit this to master branch.

Reply:
removed in latest commit